OrbitCore/OrbitProcess.cpp

```diff
@@ -536,7 +536,8 @@ void Process::FindCoreFunctions() {
 //-----------------------------------------------------------------------------
 ORBIT_SERIALIZE(Process, 2) {
   ORBIT_NVP_VAL(0, m_Name);
-  ORBIT_NVP_VAL(0, m_FullName);
+  ORBIT_NVP_VAL(0, m_FullPath);
```

Comment:
When changing a variable name, or adding/removing fields, you should increase the version number, and use that new version in the ORBIT_NVP_VAL macro, this will enable backwards compatibility. 

ORBIT_SERIALIZE(Process, 3) {
ORBIT_NVP_VAL(0, m_Name);  
ORBIT_NVP_VAL(3, m_FullPath);
ORBIT_NVP_VAL(3, m_CmdLine);

Reply:
Ah makes sense, done.